README.md

```diff
@@ -4,6 +4,24 @@ easy-schema
 A collection of XML schema's used by EASY. These schema's are published at 
 [https://easy.dans.knaw.nl/schemas/](https://easy.dans.knaw.nl/schemas/)
 
+
+Maintain sword-v1-packaging
+---------------------------
+
+The Sword-v1 [service document] (ignore login) refers to the [packaging document]
```

Comment:
Have you tested these links? I'm pretty sure the first one doesn't. At least, not for me when I click `view` and try them...